cassandra/src/main/scala/filodb.cassandra/columnstore/CassandraColumnStore.scala

```diff
@@ -78,9 +81,10 @@ extends ColumnStore with CassandraChunkSource with StrictLogging {
                             MeasurementUnit.time.milliseconds).withoutTags()
 
   def initialize(dataset: DatasetRef, numShards: Int): Future[Response] = {
+    if (createTablesEnabled) {
       val chunkTable = getOrCreateChunkTable(dataset)
       val partitionKeysByUpdateTimeTable = getOrCreatePartitionKeysByUpdateTimeTable(dataset)
-    if (createTablesEnabled) {
+      val partitionKeysV2Table = getOrCreatePartitionKeysV2Table(dataset)
```

Comment:
Note that `getOrCreatePartitionKeysV2Table` does not actually create the table in cassandra. It creates the `PartitionKeysV2Table` object. The `initialize` call will create the tables, and is lazy.

I did not add the guards for v1 tables because we did not narrow down on migration strategy. We may choose to migrate with co-existing tables. That is an option on the table.

Nevertheless, it is likely we may not choose the co-existing tables strategy for migration, so I will disable the table object initialization anyway.